README.md

```diff
@@ -177,6 +177,77 @@ host    all             all             127.0.0.1/32            md5
 sudo systemctl start redis memcached rabbitmq-server postgresql
 ```
 
+On CentOS 7 (Core):
+
+```
+# Add user zulip to the system (not necessary if you configured zulip as the administrator
+# user during the install process of CentOS 7).
+useradd zulip
+
+# Create a password for zulip user
+passwd zulip
+
+# Allow zulip to sudo
+visudo
+# Add this line after line `root    ALL=(ALL)       ALL`
+zulip   ALL=(ALL)       ALL
+
+# Switch to zulip user
+su zulip
+
+# Enable EPEL 7 repo so we can install rabbitmq-server, redis and other dependencies
+sudo yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+
+# Install dependencies
+sudo yum install libffi-devel memcached rabbitmq-server openldap-devel python-devel redis postgresql-server \
+postgresql-devel postgresql libmemcached-devel wget python-pip openssl-devel freetype-devel libjpeg-turbo-devel \
+zlib-devel nodejs
+
+# We need these packages to compile tsearch-extras
+sudo yum groupinstall "Development Tools"
+ 
+cd && wget https://launchpad.net/~tabbott/+archive/ubuntu/zulip/+files/tsearch-extras_0.1.3.tar.gz
+tar xvzf tsearch-extras_0.1.3.tar.gz
+cd ts2
+make -j $(nproc)
+sudo make install
+
+# Now create the missing dictionary files
+sudo touch /usr/share/pgsql/tsearch_data/zulip_english.stop
+sudo touch /usr/share/pgsql/tsearch_data/english.stop
+sudo touch /usr/share/pgsql/tsearch_data/en_us.dict
+sudo touch /usr/share/pgsql/tsearch_data/en_us.affix
```

Comment:
This should probably include the comment we have on the Fedora instructions about how this piece is not totally correct and will result in full-text search not working (until we fix this).